src/main/java/main/RestManager.java

```diff
@@ -3,10 +3,9 @@
 import controller.OfferManager;
 import model.Offer;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestBody;
```

Comment:
I suggest to leave these imports as they were. We are new to Spring so we want to know which Spring classes we are using exactly.